iloop_to_model/app.py

```diff
@@ -53,6 +53,50 @@ class Meta:
         return OrganismToTaxonMessage(ILOOP_SPECIES_TO_TAXON)
 
 
+def group_id(sample):
+    """Unique identifier for the sample using its properties"""
+    return (
+        sample.strain.pool.id,
```

Comment:
that's a valid concern, but in this case no - the thing about fermentation experiment according to Emre that the distinction is actually in the pool, and all the strains in one pool have different identifiers but the same genotype. So using strain ids would only lead to not grouping samples at all